tests/test_cli.py

```diff
@@ -477,15 +470,14 @@ def test_handler_domain_1(self, fake_load_dotenv_1):
             parse_env()
             console = Console()
             progress = (simulate_progress(console),)
-            entity = generate_entity_handler(parse_args(), console, progress)
+            entity = generate_entity_handler(parse_args(), console, progress)  # type: ignore
```

Comment:
`# type: ignore` is not really necessary on test files because we only run mypy against `wtfis/` (not `tests/`) during build time - see [this](https://github.com/pirxthepilot/wtfis/blob/9bcc55bceb513d262c04edc104ee3d1278b19e72/pyproject.toml#L74)